core/block_svg.ts

```diff
@@ -244,9 +244,12 @@ export class BlockSvg
 
   private computeAriaLabel(): string {
     const {blockSummary, inputCount} = buildBlockSummary(this);
-    const inputSummary = inputCount
-      ? ` ${inputCount} ${inputCount > 1 ? 'inputs' : 'input'}`
-      : '';
+    let inputSummary = '';
+    if (inputCount > 1) {
+      inputSummary = 'has inputs';
+    } else if (inputCount === 1) {
+      inputSummary = 'has input';
```

Comment:
we are going to be sad later because we don't support plural translations :/

(no action needed, this is just me wishing we had more robust i18n systems)